core/src/components/content/content.tsx

```diff
@@ -307,6 +307,7 @@ export class Content implements ComponentInterface {
     const { scrollX, scrollY } = this;
     const mode = getIonMode(this);
     const forceOverscroll = this.shouldForceOverscroll();
+    const TagType = this.el.closest('ion-menu') !== null ? 'div' : 'main' as any;
```

Comment:
```suggestion
    const TagType = this.el.closest('ion-menu, ion-popover, ion-modal') !== null ? 'div' : 'main' as any;
```

We should account for this in popovers and modals as well since those are not "main" landmarks/they are temporary.